x-pack/plugins/ml/public/application/jobs/new_job/pages/job_type/page.tsx

```diff
@@ -217,8 +217,8 @@ export const Page: FC = () => {
             <p>
               <FormattedMessage
                 id="xpack.ml.newJob.wizard.jobType.useSuppliedConfigurationDescription"
-                defaultMessage="The fields in your data have been recognized as matching known configurations.
-                Select to create a set of machine learning jobs and associated dashboards."
+                defaultMessage="The fields in your data match known configurations.
+                Create a set of preconfigured jobs and associated dashboards."
```

Comment:
I wonder if now is a good time to drop the `and associated dashboards` part of the sentence? Only 4 out of the current 18 modules actually create Kibana dashboards.